src/test/java/org/folio/consortia/domain/EntityTest.java

```diff
@@ -0,0 +1,68 @@
+package org.folio.consortia.domain;
+
+import org.folio.consortia.domain.entity.TenantEntity;
+import org.folio.consortia.domain.entity.UserTenantEntity;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+class EntityTest {
```

Comment:
Do we really need tests to test entities?

Reply:
Not really needed but just to be on safer side.